addons/pos_hr/static/src/overrides/components/navbar/navbar.xml

```diff
@@ -7,16 +7,24 @@
                 !pos.config.module_pos_hr or employeeIsAdmin()
             </attribute>
         </xpath>
-        <xpath expr="//DropdownItem[contains(text(), 'Backend')]" position="after">
-            <DropdownItem t-if="pos.config.module_pos_hr" onSelected="() => this.showLoginScreen()">
-                Lock
-            </DropdownItem>
-        </xpath>
         <xpath expr="//DropdownItem[contains(text(), 'Close Register')]" position="attributes">
             <attribute name="t-if">
                 !pos.config.module_pos_hr or employeeIsAdmin()
             </attribute>
         </xpath>
+        <xpath expr="//DropdownItem[contains(text(), 'Create Product')]" position="attributes">
+            <attribute name="t-if">
+                !pos.config.module_pos_hr or employeeIsAdmin()
+            </attribute>
+        </xpath>
+        <xpath expr="//DropdownItem[contains(text(), 'Edit Plan')]" position="attributes">
+            <attribute name="t-if">
+                !pos.config.module_pos_hr or employeeIsAdmin()
+            </attribute>
+        </xpath>
+        <xpath expr="//CashierName" position="after">
+            <button t-if="pos.config.module_pos_hr" class="btn btn-light fa fa-lg fa-lock text-success px-3" title="Lock" t-on-click="showLoginScreen"/>
+        </xpath>
```

Comment:
This xpath override is unwieldy. I'd rather introduce a method in point_of_sale that checks the visibility of the buttons, then override the method here in pos_hr.